app/interactions/create_match_for_contribution.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class CreateMatchForContribution < ActiveInteraction::Base
```

Comment:
I'd consider naming this `MatchContribution` because in addition to creating a match, it also marks the existing contribution as 'matched'.

Reply:
Renamed the file